dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/publish/servers/com.ibm.ws.jaxws.virtualhost/bootstrap.properties

```diff
@@ -1,5 +1,5 @@
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
 com.ibm.ws.jaxws.*=all=enabled
 com.ibm.ws.logging.max.file.size=0
-
+websphere.java.security.exempt=true
```

Comment:
Why are you doing this?

Reply:
The virtualhost Java 2 Sec errors are occurring in a non-Web Services Sun RI class, and rest of the stack in the failure is completely outside of the Web Services. I initially tried to add the Sun property permission to the server.xml, but it had no effect. I realize this is a using a bandage rather than a band-aid, but if the Java 2 sec failures in Web Services code occurred here, it would also should be occurring in the other tests if you're worried about this hiding other failures.

Reply:
Yeah the server.xml also gives all permissions to everything so I thought it was rather odd to be required.